samples/drivers/espi/prj_npcx7m6fb_evb.conf

```diff
@@ -0,0 +1,10 @@
+# eSPI + npcx7m6fb_evb
+CONFIG_LOG=y
+CONFIG_LOG_BUFFER_SIZE=4096
+CONFIG_LOG_PROCESS_THREAD_SLEEP_MS=100
+
+# EC Host Command Driver
+CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD=y
```

Comment:
We shouldn't have this config option yet (since it isn't implement yet, right)?
```suggestion
```

Reply:
Oops. I miss this line when I dropped the implementation CL. Thanks.

Reply:
Done.